vcap/vcap/ovens.py

```diff
@@ -25,17 +25,24 @@ class Oven:
     running that work in a batched predict function, then returning that
     work to the respective output queues."""
 
-    MAX_BATCH_SIZE = 40
-
     def __init__(self, batch_fn: Callable[[List[Any]], List[Any]],
+                 max_batch_size=40,
```

Comment:
I think the default should still be defined by a constant

Reply:
As far as defining a default, I think keyword arguments are the best way to go right now. There is no other location someone would want to use this number.

Reply:
I think you might be misunderstanding me. I think it should be `max_batch_size=MAX_BATCH_SIZE` in the args list

Reply:
No, I understand you. I just think that the '40' should live there, not in a constant. I don't see the point of adding a constant, because it implies more importance to an otherwise arbitrary number.